terraform/alert_policy/missing_metric/main.tf

```diff
@@ -0,0 +1,23 @@
+resource "google_monitoring_alert_policy" "missing_metric" {
+  display_name = "${var.display_name} (branch suffix: ${var.branch_suffix})"
```

Comment:
Maybe just pass the full display_name as a variable to make it consistent with line 6?